views/js/previewer/provider/item/scss/inc/_previewer-layout.scss

```
@@ -22,12 +22,16 @@
         height: calc(100vh - #{(map-get($heights, header) + map-get($heights, footer)) * 1px});
 
         .test-sidebar {
-            height: calc(100vh - #{(map-get($heights, header) + map-get($heights, footer) + map-get($heights, horizontal-action-bar) * 2) * 1px});
+            height: calc(
+                100vh - #{(
+                        map-get($heights, header) + map-get($heights, footer) + map-get($heights, horizontal-action-bar) *
```

Stylelint (17.14.1)

[error] Unexpected newline after "*" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)